src/hipscat/pixel_math/pixel_margins.py

```diff
@@ -37,71 +38,76 @@ def get_edge(d_order, pix, edge):
+        one-dimensional numpy array of long integers, filled with the healpix pixels
             at order kk+dk that border the given edge of pix.
     """
     if edge < 0 or edge > 7:
         raise ValueError("edge can only be values between 0 and 7 (see docstring)")
-        
-    if (edge, d_order) in _suffixes.keys():
+
+    if (edge, d_order) in _suffixes:
         pixel_edge = _suffixes[(edge, d_order)]
     else:
         # generate and cache the suffix:
 
         # generate all combinations of i,j,k,... suffixes for the requested edge
         # See https://stackoverflow.com/a/35608701
-        grid = np.array(np.meshgrid(*[_edge_vectors[edge]]*d_order))
+        grid = np.array(np.meshgrid(*[_edge_vectors[edge]] * d_order))
         pixel_edge = grid.T.reshape(-1, d_order)
         # bit-shift each suffix by the required number of bits
-        pixel_edge <<= np.arange(2*(d_order-1),-2,-2)
+        pixel_edge <<= np.arange(2 * (d_order - 1), -2, -2)
         # sum them up row-wise, generating the suffix
         pixel_edge = pixel_edge.sum(axis=1)
         # cache for further reuse
         _suffixes[(edge, d_order)] = pixel_edge
 
     # append the 'pix' preffix
-    pixel_edge = (pix << 2*d_order) + pixel_edge
+    pixel_edge = (pix << 2 * d_order) + pixel_edge
     return pixel_edge
 
+
 def get_margin(order, pix, d_order):
     """Get all the pixels at order order+dk bordering pixel pix.
     See hipscat/pixel_math/README.md for more info.
 
     Args:
         order (int): the healpix order of pix.
         pix (int): the healpix pixel to find margin pixels of.
-        d_order (int): the change in k that we wish to find the margins for. 
+        d_order (int): the change in k that we wish to find the margins for.
             Must be greater than kk.
     Returns:
-        one-dimensional numpy array of long integers, filled with the healpix pixels 
+        one-dimensional numpy array of long integers, filled with the healpix pixels
             at order kk+dk that border pix.
     """
     if d_order < 1:
         raise ValueError("dk must be greater than order")
-    
+
     nside = hp.order2nside(order)
 
     # get all neighboring pixels
     neighbors = hp.get_all_neighbours(nside, pix, nest=True)
 
     # get the healpix faces IDs of pix and the neighboring pixels
     _, _, pix_face = hp.pix2xyf(nside, pix, nest=True)
-    _, _, faces  = hp.pix2xyf(nside, neighbors, nest=True)
+    _, _, faces = hp.pix2xyf(nside, neighbors, nest=True)
 
     # indices which tell get_edge() which edge/verted to return
     # for a given pixel. The default order is compatible with the
     # order returned by hp.get_all_neighbours().
     which = np.arange(8)
-    if pix_face < 4: # northern hemisphere; 90deg cw rotation for every +1 face increment
-        mask = (faces < 4)
-        which[mask] += 2*(faces-pix_face)[mask]
+    if (
+        pix_face < 4
+    ):  # northern hemisphere; 90deg cw rotation for every +1 face increment
```

Comment:
As comment from the side: This formatting doesn't seem to help the readability of the if statement. Is it being broken up over three lines because the comment is on the same line?

Reply:
Ewww. `black` did gross things.